legal-api/migrations/versions/24b59f535ec3_add_currency_additional_to_share_classes.py

```diff
@@ -0,0 +1,26 @@
+"""add_currency_additional_to_share_classes
+
+Revision ID: 24b59f535ec3
+Revises: f1d010259785
+Create Date: 2025-02-28 23:28:54.053129
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '24b59f535ec3'
+down_revision = 'f1d010259785'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('share_classes', sa.Column('currency_additional', sa.String(length=40)))
```

Comment:
Fix `psycopg2.errors.StringDataRightTruncation) value too long for type character varying(10)` - Part 1.

COLIN has free format text input for currency, which exceed the limit of characters in LEAR.